boards/xtensa/intel_s1000_crb/support/download.py

```diff
@@ -0,0 +1,103 @@
+#!/usr/bin/env python3
+#
+# Copyright (c) 2019 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+# Author: Sathish Kuttan <sathish.k.kuttan@intel.com>
+
+import os
+import sys
+import hashlib
+import device
+import messenger
+
+sue_creek = device.Device()
+msg = messenger.Message()
+
+def check_arguments():
+    if len(sys.argv) != 2:
+        print('Usage: python3 %s <zephyr.bin>' % sys.argv[0])
```

Comment:
While you're writing this out, including a one line "what the script is for" help would be good too.

Reply:
I added it in the file below the header.